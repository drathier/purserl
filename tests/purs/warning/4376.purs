-- @shouldWarnWith MissingTypeDeclaration
module Main where

data Maybe a = Just a | Nothing

value = Nothing
