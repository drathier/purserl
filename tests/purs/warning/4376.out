Warning found:
in module [33mMain[0m
at tests/purs/warning/4376.purs:6:1 - 6:16 (line 6, column 1 - line 6, column 16)

  No type declaration was provided for the top-level declaration of [33mvalue[0m.
  It is good practice to provide type declarations as a form of documentation.
  The inferred type of [33mvalue[0m was:
  [33m                    [0m
  [33m  forall @a. Maybe a[0m
  [33m                    [0m

in value declaration [33mvalue[0m

See https://github.com/purescript/documentation/blob/master/errors/MissingTypeDeclaration.md for more information,
or to contribute content related to this warning.

